Releasenotes.md

```diff
@@ -1,6 +1,76 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
+  - Er is gebruik gemaakt van Haal-Centraal-common versie 1.2.0 (was 1.0.0)
```

Comment:
Het is volgens mij toch echt `gebruikgemaakt`, heb het nagezocht ([onzetaal](https://onzetaal.nl/taaladvies/gebruikmaken-gebruik-maken/#close)) 😛